back/db/migrate/20240722090955_remove_map_configs_without_mappable.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# This is a data migration. It does not change the schema.
+class RemoveMapConfigsWithoutMappable < ActiveRecord::Migration[7.0]
+  class MapConfig < ApplicationRecord
+    self.table_name = 'maps_map_configs'
+  end
+
+  class Layer < ApplicationRecord
+    self.table_name = 'maps_layers'
+
+    belongs_to :map_config, class_name: 'MapConfig'
+  end
+
+  def change
+    map_configs = MapConfig.where(mappable_id: nil, mappable_type: nil)
+    layers = Layer.where(map_config: map_configs)
```

Comment:
Do we have an idea of the max number of map configs per tenant? Worried that the layers query could end up being very large if there are millions in one tenant for example as I think rails will convert this into a `where id IN ('id1','id2','etc')` query

Reply:
No, it should in principle use a nested query in that case. Also, I removed the records manually for the biggest one, and there were actually 0 layers to be removed.